var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -51,9 +51,13 @@ class Mfem(Package):
     # If this quick verification procedure fails, additional discussion
     # will be required to verify the new version.
 
+    version('3.3.1-rc1', git='https://github.com/mfem/mfem',
+            tag='v3.3.1-rc1')
+
     version('3.3',
             'b17bd452593aada93dc0fee748fcfbbf4f04ce3e7d77fdd0341cc9103bcacd0b',
-            url='http://goo.gl/Vrpsns', extension='.tar.gz')
+            url='http://goo.gl/Vrpsns', extension='.tar.gz',
+            preferred=True)
```

Comment:
Note that `version` lines don't have to be < 80 chars if you don't want them to be.

Reply:
Is there a reason to not download them from https://github.com/mfem/mfem/tags?

Reply:
Yes, we collect statistics on the tarball links using the goo.gl shortened links.